def nested_exceptions(tree):
     = 889
    return max

print(nested_exceptions(2))